build.gradle.kts

```diff
@@ -132,10 +153,10 @@ tasks.named<ShadowJar>("shadowJar") {
     archiveClassifier.set("")
     archiveBaseName.set("Terra")
     setVersion(project.version)
-    relocate("org.apache.commons", "com.dfsek.terra.lib.commons")
```

Comment:
These classes should be relocated to inside the Terra package, to avoid potential conflicts with other plugins.